### PR TITLE
i18n(es): update `authoring-content`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updates `authoring-content` based on this commit 1a9ab50d458d6274994ffe66a23fe7a30681337a

### CODE REVIEW

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 1]
```diff
@@ -360,6 +360,31 @@ Un título opcional del bloque de código se puede establecer con un atributo `t
   ```
   ````
 
+## Details
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/authoring-content.md:363]: Just to double check, is it expected to have the same heading as the English version or should this be translated too like the other headings?

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left 1 tiny question but except for that, everything looks perfect to me.

File: docs/src/content/docs/es/guides/authoring-content.md (hunk 1 above)
Comment (dreyfus92) [reply docs/src/content/docs/es/guides/authoring-content.md:363]: Yeah, as far as I know it is. I went to MDN's docs for it. Here is the [example]( https://developer.mozilla.org/es/docs/Web/HTML/Element/details)

**Review by HiDeoo — APPROVED** [review]
Thanks for the clarification :thumbsup:

**Pull request merged**